org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "org.js,nuclear.Nuclear",
```

Comment:
you put a comma instead of a period.

Reply:
yeah sorry its pretty late at night for me.